src/core/tests/model.cpp

```diff
@@ -1334,6 +1334,63 @@ bool all_ops_have_same_info(const std::shared_ptr<ov::Model>& f) {
 }
 }  // namespace
 
+TEST(model, topological_sort_throws_if_loop_with_one_node) {
```

Comment:
please check how we report the case when loop exists in Loop(TI, If) ops, is our message informative enough to undestand the root cause?

Reply:
Added test for Loop. Error message points to direct layer, not to Loop

Reply:
I decided to remove the test - since Loop requires body which is ov::Model, test fails on body creation. So there is no way to create Loop with cycled ov::Model